sample/android/build.gradle.kts

```diff
@@ -7,11 +7,11 @@ plugins {
 
 android {
     namespace = "co.yml.ychat.android"
-    compileSdk = Config.COMPILE_SDK_VERSION
+    compileSdk = libs.versions.config.compile.sdk.version.get().toInt()
     defaultConfig {
         applicationId = "co.yml.ychat.android"
-        minSdk = Config.MIN_SDK_VERSION
-        targetSdk = Config.TARGET_SDK_VERSION
+        minSdk = libs.versions.config.min.sdk.version.get().toInt()
```

Comment:
Having to manually cast to int is a code smell IMO, but this seems to rely in the inability of Gradle to do it automatically.

Reply:
@kikoso Good call out! Unfortunately Gradle doesn't doe it automatically and Version Catalog only supports strings, but I've seen that a recent common practice seems to be holding these SDK versions on `build-logic` sub module so I think we can do that in the future too as it'll be useful for our plugins and custom scripts.